ui/app/adapters/pki/action.js

```diff
@@ -7,7 +7,7 @@ export default class PkiActionAdapter extends ApplicationAdapter {
 
   urlForCreateRecord(modelName, snapshot) {
     const { type } = snapshot.record;
-    const { actionType, useIssuer, issuerName, mount } = snapshot.adapterOptions;
+    const { actionType, useIssuer, issuerRef, mount } = snapshot.adapterOptions;
```

Comment:
Could we get this directly from `snapshot.record` instead?

Reply:
Good question! The model being saved here is a `pki/action` to sign a CSR - so the `issuerRef` is coming from the parent mount where the signing is happening and isn't on the record. 

Another option is to add this attribute to the pki/action model, and pass it to the `createRecord` instead of the `save()` method [here](https://github.com/hashicorp/vault/blob/main/ui/lib/pki/addon/components/pki-issuer-cross-sign.js#L138-L150) but this feels a little wonky since it's not a param in the payload, and is part of the request URL ([sign-intermediate docs](https://developer.hashicorp.com/vault/api-docs/secret/pki#parameters-3))

I think keeping it as is aligns more with what happening - but what do you think? I've stared at this file way too much 🥴